frontend/packages/pipelines-plugin/src/components/pipelines/modals/common/__tests__/VolumeClaimTemplateForm.spec.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { formikFormProps } from '@console/shared/src/test-utils/formik-props-utils';
+import { ExpandCollapse, RequestSizeInput } from '@console/internal/components/utils';
+import { RadioInput } from '@console/internal/components/radio';
+import VolumeClaimTemplateForm from '../VolumeClaimTemplateForm';
+
+let volumeClaimTemplateFormProps: React.ComponentProps<typeof VolumeClaimTemplateForm>;
+
+jest.mock('react-i18next', () => {
+  const reactI18next = require.requireActual('react-i18next');
+  return {
+    ...reactI18next,
+    useTranslation: () => ({ t: (key) => key }),
+  };
+});
+
+jest.mock('formik', () => ({
+  useField: jest.fn(() => [{}, {}]),
+  useFormikContext: jest.fn(() => ({
+    setFieldValue: jest.fn(),
+    setFieldTouched: jest.fn(),
+    validateForm: jest.fn(),
+  })),
+  getFieldId: jest.fn(),
+}));
+
+describe('VolumeClaimTemplateForm', () => {
+  beforeEach(() => {
+    volumeClaimTemplateFormProps = {
+      ...formikFormProps,
+      name: 'test.form',
+    };
+  });
+
+  it('should render correct HelpText and ExpandCollapse', () => {
```

Comment:
should we be testing if a component renders unless it is rendered conditionally or the number of instances of a component rendered, changes?